src/components/TimeseriesDataExport/TimeseriesDataExport.tsx

```diff
@@ -36,31 +31,15 @@ import {
 type TimeseriesDataExportFormProps = TimeseriesDataExportProps &
   FormComponentProps;
 
-// TODO: Check tree shacking for TimeseriesDataExport component
+// TODO: Check tree shaking for TimeseriesDataExport component
 const { RangePicker } = DatePicker;
 const CELL_LIMIT = 10000;
-const formatData = 'YYYY-MM-DD HH:mm:ss';
+const formatData = 'YYYY-MM-DD_HH:mm:ss';
```

Comment:
I'm guessing this change was on purpose, but could you perhaps satisfy my curiosity as to why it was made? :)

Reply:
space breaks CSV formatting for different viewers, MS Excel for instance